### DISCUSSION TITLE
Last Commit on 13-March-2022

### DISCUSSION THREAD

**Discussion by chrisjbreisch** [WhitewaterFoundry/Pengwin#746] [General]
Should I be concerned that the last commit on this project was on the 13th of March, 2022, and that was over 4 months ago?

**Comment by crramirez** [comment]
Hello @chrisjbreisch,

The project is fully alive, and we are fine-tuning a new feature. As a small team, sometimes we need to focus on other projects entirely, but if a bug arises, depending on the severity will be attended immediately. 

Don't worry, Pengwin is fully functional and stable nowadays.

Regards,
Carlos

**Reply by crramirez** [reply]
Also, I forgot to mention that Pengwin comprises four projects in GitHub:

1. This one that holds the application and packaging to the Store. Only when we publish a new version to the store or add new functionality to the launcher, you'll see commits here.
2. https://github.com/WhitewaterFoundry/pengwin-base holds the base scripts for Pengwin, and it only gets changed with a new version or when we change the core.
3. https://github.com/WhitewaterFoundry/pengwin-setup the installed for Pengwin, you'll see most of the commits happens in this repo as we constantly adjust the scripts or add new ones.
4. https://github.com/WhitewaterFoundry/pengwin-rootfs-builds where the image creation happens, we modify it only when we make a structural change in Pengwin.

You can see changes besides these repos when we backport and rebuild packages from mesa, xrdp, and glibc to include patches that belong to WSL only.
